### DISCUSSION TITLE
Is it possible to add audio effect in Oboe Live stream

### DISCUSSION THREAD

**Discussion by YogarajRamesh** [google/oboe#1508] [Q&A]
Is it possible to add audio effect in oboe:audioStream
(already i did live feedback and record voice. now i need to add echo effects to the stream )

Is it possible to add fxlab effects to LiveEffectEngine?

**Comment by philburk** [comment]
The source for FXLab is available. You are free to use that in your application.

We cannot advise you on the details of how to do that.
